### DISCUSSION TITLE
British English

### DISCUSSION THREAD

**Discussion by MattMurray06** [monkeytypegame/monkeytype#1302] [Ideas] [answered]
Hey, just wondering if British English could be added to the list of languages?
I realise [almost typed realize!] that there isn't much difference but it would be great to have the option.
Thanks!
(P.S please tell me if this is in the wrong section, it's my first time on GitHub!)

**Comment by Banzobotic** [comment, marked as answer]
There is a pr for this at #958 but Miodec suggested that this be added as a setting rather than as a second language. Afaik this is just waiting for someone to implement it.

**Reply by MattMurray06** [reply]
Ah, didn't see that. Thanks
